opentelemetry-sdk/benches/log.rs

```diff
@@ -27,6 +28,12 @@ fn log_benchmark_group<F: Fn(&dyn Logger)>(c: &mut Criterion, name: &str, f: F)
 
     group.bench_function("no-context", |b| {
         let provider = LoggerProvider::builder()
+            .with_config(Config::default().with_resource(Resource::new(vec![
```

Comment:
https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry-sdk/src/logs/config.rs#L19-L20

Wondering why config stores `Resource` inside a `Cow` ?

Reply:
It's to avoid performance overhead. `Cow` provides cheap copy until the data changes or an ownership is required